doc/release_instructions.md

```diff
@@ -79,20 +94,20 @@ nothing added to commit but untracked files present (use "git add" to track)
 > git push origin master
 ```
 
-8) Regenerate the golden deployment manifests: `make regenerate-test-deploy-manifests`
+9) Regenerate the golden deployment manifests: `make regenerate-test-deploy-manifests`
 
-9) Commit the changes:
+10) Commit the changes:
 
 ```
 > git commit -a -m"Regenerate golden deployment manifests for $(pachctl version --client-only)"
 > git push origin master
 ```
 
-10) Update the
```

Comment:
If we're reconsidering the order of these steps - I'd propose we move the changelog up above the `make point-release` step.  Obviously we can't edit the release notes, but it's rather nice to have the `CHANGELOG.md` contain the relevant data under the release's git tag.